docs/sources/writing-guide/contribute-documentation/index.md

```diff
@@ -105,8 +129,7 @@ When you are ready for other people to review your work, perform the following t
 1. Commit your changes.
 1. Push your changes to Github.
 1. Create a PR in Github.
-1. When writing the description for your PR, use [GitHub keywords](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue), for example "Fixes #1234", to link your pull request to the issue and take advantage of GitHub automation for status updates and closing resolved issues.
-1. Add the `type/docs` label, so the Docs team can track the issue.
+1. Add the `type/docs` label.
```

Comment:
I prefer to keep this rationale in here. Alternatively, and below this line, we could add instead:

"The Docs team uses the `type/docs` label across all Grafana Labs project and products to manage incoming documentation issues."